### ISSUE TITLE
no validation set

### ISSUE THREAD

**Issue by PaulJWright** [yanx27/Pointnet_Pointnet2_pytorch#91]
You're not using a validation set, but instead are using the test set for hyperparameter tuning?
